CHANGE_LOG

```diff
@@ -1,3 +1,168 @@
+Version 0.46.0
+--------------
+
+This release significantly reworked one of the main parts of Numba, the compiler
+pipeline, to make it more extensible and easier to use. The purpose of this was
+to continue enhancing Numba's ability for use as a compiler toolkit. In a
+similar vein, Numba now has an extension registration mechanism to allow other
+Numba-using projects to automatically have their ``@jit`` decorated functions
```

Comment:
>  have their ``@jit`` decorated functions

I am afraid one may interpret this as only ``@jit`` decorated functions are discovered

Reply:
6215c18  fixes